CONTRIBUTING.md

```diff
@@ -110,6 +110,8 @@ The following is a list of best practices to consider when writing code for the
 
 * Inline comments explaining complex codes and methods are appreciated.
 
+* Capitalize Parsons
```

Comment:
@ehrenfoss Thanks for working through this documentation stuff. Could you qualify this to say something about how contributors should capitalize Parsons in documentation and comments?